apiv1/models/CourseModel.js

```diff
@@ -4,7 +4,17 @@ let Schema = mongoose.Schema;
 let LessonSchema = new Schema({
 	'name': { type: String, required: true },
 	'prompt': String,
-	'code': { type: String, required: true }
+	'code': { type: String, required: true },
+    'settings': {
```

Comment:
Since this is going to be used across 2 different schemas, it makes sense to have the settings structure be in a file that `LessonSchema` and `SceneSchema` use.  That way we won't have issues where new additions to the schema doesn't get applied to one or the other.